3.6/config.sh

```diff
@@ -23,3 +23,31 @@ if [ ! -z "$POSTGRES_SYNCHRONOUS_COMMIT" ]; then sed -i "s/synchronous_commit =
 if [ ! -z "$POSTGRES_MAX_WAL_SIZE" ]; then sed -i "s/max_wal_size = 1GB/max_wal_size = $POSTGRES_MAX_WAL_SIZE/g" /etc/postgresql/12/main/conf.d/postgres-tuning.conf; fi
 if [ ! -z "$POSTGRES_CHECKPOINT_TIMEOUT" ]; then sed -i "s/checkpoint_timeout = 10min/checkpoint_timeout = $POSTGRES_CHECKPOINT_TIMEOUT/g" /etc/postgresql/12/main/conf.d/postgres-tuning.conf; fi
 if [ ! -z "$POSTGRES_CHECKPOINT_COMPLETITION_TARGET" ]; then sed -i "s/checkpoint_completion_target = 0.9/checkpoint_completion_target = $POSTGRES_CHECKPOINT_COMPLETITION_TARGET/g" /etc/postgresql/12/main/conf.d/postgres-tuning.conf; fi
+
+# import style tuning
+if [ "$IMPORT_STYLE" != "" ]; then
+
+    case $IMPORT_STYLE in
+    
+      admin)
+        echo "Use admin import style"
+        sed -i "s/import-full.style/import-admin.style/g" /app/src/build/settings/local.php
+        ;;
+    
+      street)
+        echo "Use street import style"
+        sed -i "s/import-full.style/import-street.style/g" /app/src/build/settings/local.php
+        ;;
+    
+      address)
+        echo "Use address import style"
+        sed -i "s/import-full.style/import-address.style/g" /app/src/build/settings/local.php
+        ;;
+        
+      extratags)
+        echo "Use extratags import style"
+        sed -i "s/import-full.style/import-extratags.style/g" /app/src/build/settings/local.php
+        ;;
+    esac
+else
+    echo "Use full import style"
```

Comment:
I also think that there is a `fi` missing at the end. Check for example https://linuxhandbook.com/if-else-bash/ for the correct syntax.